nucypher/cli/main.py

```diff
@@ -197,31 +202,38 @@ def ursula(click_config,
     #
     # Boring Setup Stuff
     #
-    log = Logger('ursula.cli')
+    if not quiet:
+        log = Logger('ursula.cli')
 
-    if debug:
+    if debug and not quiet:
         click_config.log_to_sentry = False
         click_config.log_to_file = True
         globalLogPublisher.removeObserver(logToSentry)                          # Sentry
         globalLogPublisher.addObserver(SimpleObserver(log_level_name='debug'))  # Print
 
+    elif quiet:
+        globalLogPublisher.removeObserver(logToSentry)
```

Comment:
This is neat.

Reply:
Hopefully it's enough - I actually on worried about the possibility of lingering observers, so there is *no* guarantee that logging is disabled at the moment.